_posts/2016-5-1-this-week-in-rust-docs-2.md

```diff
@@ -0,0 +1,54 @@
+---
+layout: post
+title: This Week in Rust Docs 1
+date: 2016-05-01
+---
+
+Hello and welcome to *This Week in Rust Docs*!
+
+*This Week in Rust Docs* is openly developed [on GitHub](https://github.com/GuillaumeGomez/this-week-in-rust-docs).
+If you find any errors in this week's issue, [please submit a PR](https://github.com/GuillaumeGomez/this-week-in-rust-docs/pulls).
+
+This week's edition was edited by: [GuillaumeGomez](https://github.com/GuillaumeGomez).
+
+# Latest news
+
+Things are moving on: we talked about hosting crates' documentation directly on an official doc website (the hostname hasn't been decided yet). A lot of details must be taken in account and it is still at a very early stage of discussion but it started!
```

Comment:
I might mention that this has been discussed a few times at in the past, as well: we're mostly trying to actually sort through all of the complexity here.